ui/src/components/view/ListView.vue

```diff
@@ -240,6 +240,12 @@
       <template v-if="column.key === 'agentstate'">
         <status :text="text ? text : ''" displayText />
       </template>
+      <template v-if="column.key === 'cpunumber'">
+        <span>{{ record.serviceofferingdetails ? `${record.serviceofferingdetails.mincpunumber} - ${record.serviceofferingdetails.maxcpunumber}` : record.cpunumber }}</span>
```

Comment:
```suggestion
        <span>{{ record.serviceofferingdetails?.mincpunumber && record.serviceofferingdetails?.maxcpunumber ? `${record.serviceofferingdetails.mincpunumber} - ${record.serviceofferingdetails.maxcpunumber}` : record.cpunumber }}</span>
```

This results in an error when `serviceofferingdetails` contains details other than `mincpunumber` and `maxcpunumber`, such as when the compute offering is associated to a zone. We should check if both `mincpunumber` and `maxcpunumber` are present instead.

![image](https://github.com/user-attachments/assets/1ddc34d2-d789-4be6-870c-71aa3c251d4b)